lib/feature.pm

```diff
@@ -29,7 +30,7 @@ our %feature_bundle = (
     "5.15"    => [qw(current_sub evalbytes fc say state switch unicode_eval unicode_strings)],
     "5.23"    => [qw(current_sub evalbytes fc postderef_qq say state switch unicode_eval unicode_strings)],
     "5.27"    => [qw(bitwise current_sub evalbytes fc postderef_qq say state switch unicode_eval unicode_strings)],
-    "all"     => [qw(bitwise current_sub declared_refs evalbytes fc postderef_qq refaliasing say signatures state switch unicode_eval unicode_strings)],
+    "all"     => [qw(bitwise current_sub declared_refs evalbytes fc noindirect postderef_qq refaliasing say signatures state switch unicode_eval unicode_strings)],
```

Comment:
My first thought while reading this is the double negation, which becomes even worst when using `no noindirect`
maybe we are just talking about `use direct` instead of `use noindirect` :-)

Reply:
The problem is that it's supposed to stop you from using indirect method calls, it's not a feature you need in order to use direct ones, so calling the feature simply 'direct' seems wrong to me on that basis.

Reply:
`no feature 'indirect'` feels more logical to me

Reply:
@Leont that would imply the conceptual model that some "feature"s are on by default that you start by disabling. That changes the current idea that there are no "feature"s enabled to begin with but you can turn some of them on.

Reply:
@leonerd there's the `default` bundle just below here. it used to include `array_base` (which some of the versioned bundles didn't include) until we removed it.

Reply:
Yeah, what @ilmari said. While I do think it's a valid discussion if this is a good idea, we have done this before.